Benchmarks/Benchmarks/Predicates/Predicates.swift

```diff
@@ -12,18 +12,62 @@
 
 import Benchmark
 import func Benchmark.blackHole
+
+#if FOUNDATION_FRAMEWORK
+import Foundation
+#else
 import FoundationEssentials
+#endif
+
+let monster = Monster(name: "Orc", level: 80, hp: 100, mana: 0, weapon: .sword(Sword(p1: 1, p2: 2, p3: 3, p4: 4, p5: 5)))
+let monster2 = Monster(name: "Orc", level: 80, hp: 100, mana: 0, weapon: .sword(Sword(p1: 1, p2: 2, p3: 3, p4: 4, p5: 5)))
+
+// These tests are disabled, as enabling them will make compilation fail due to https://github.com/apple/swift/issues/69277
+#if false
+func registerPredicateTests_disabled() {
```

Comment:
Does this `_disabled()` suffix indicate other issues besides the issue 69277? Asking because  `#if false` should suffice in disabling the test, unless there's something else also going on

Reply:
No, same issue.  I was just making it a little more clear why it's inside a `if false`.